Pod/Classes/Cards/CardType.swift

```diff
@@ -42,23 +42,63 @@ public protocol CardType {
         - 9 for assignment by national standards bodies
      The first 6 digits also are the *Issuer Identification Number*, indicating the issuer of the card. 
      
-     In order to identify the card issuer, this function returns a Set of integers which indicate the card issuer. In case of Discover for example, this is the set of [(644...649),(622126...622925),(6011)], which contains different IIN ranges which are reserved for Discover.
+     In order to identify the card issuer, this function returns a Set of integers which indicate the card issuer. In case of Discover for example, this is the set of [(644...649),(622126...622925),(6011)], which contains different Issuer Identification Number ranges which are reserved for Discover.
+     
+     - important: When creating custom card types, you should make sure, that there are no conflicts in the Issuer Identification Numbers you provide. For example, using [309] to detect a Diners Club card and using [3096] to detect a JCB card will not cause issues as IINs are parsed with the highest numbers first, i.e. the numbers that provide the most context possible, which will return a JCB card in this case. However, no two card types should provide the exact same number (like [309] to detect both a Diners Club card and a JCB card)!
      
      - returns: A set of numbers which, when being found in the first digits of the card number, indicate the card issuer.
      */
     var identifyingDigits: Set<Int> { get }
 
+    /**
+     Validates the card verification code.
+     
+     - important: This validation does not mean that the CVC will be accepted by the card issuer! In order to validate the card with the card issuer, you should opt for the integration of a [Payment Gateway](https://en.wikipedia.org/wiki/Payment_gateway) to confirm the authenticity of the entered CVC.
```

Comment:
Mind if you can update the description to include that it is validating the number of digits in CVC? It looks a little bit confusing. If I am not mistaken, what it does is to inform users the invalidity of CVC when it has the wrong number of digits. E.g. for AMEX, 3 digits of CVC would result in a invalidity.